raft/src/test/java/org/apache/kafka/raft/RaftEventSimulationTest.java

```diff
@@ -53,9 +53,10 @@
 
 public class RaftEventSimulationTest {
     private static final int ELECTION_TIMEOUT_MS = 1000;
-    private static final int ELECTION_JITTER_MS = 10;
+    private static final int ELECTION_JITTER_MS = 25;
     private static final int RETRY_BACKOFF_MS = 50;
     private static final int REQUEST_TIMEOUT_MS = 500;
+    private static final int FETCH_MAX_WAIT_MS = 100;
```

Comment:
This reminds me that we need to update the KIP with this added configuration, together with the RPC change.

Reply:
I will do this.